discord/app_commands/commands.py

```diff
@@ -129,15 +129,67 @@
 VALID_SLASH_COMMAND_NAME = re.compile(r'^[\w-]{1,32}$')
 CAMEL_CASE_REGEX = re.compile(r'(?<!^)(?=[A-Z])')
 
+ARG_NAME_SUBREGEX = r"(?:\\?\*)*(?P<name>[^\s:\-]+)"
+
+ARG_DESCRIPTION_SUBREGEX = r"(?P<description>(?:.|\n)+?(?:\Z|\r?\n(?=[\S\r\n])))"
+
+ARG_TYPE_SUBREGEX = r"(?P<type>.+)"
+
+GOOGLE_DOCSTRING_ARG_REGEX = re.compile(
+    rf"^{ARG_NAME_SUBREGEX}[ \t]*(?:\({ARG_TYPE_SUBREGEX}\))?[ \t]*:[ \t]*{ARG_DESCRIPTION_SUBREGEX}",
+    re.MULTILINE,
+)
+
+SPHINX_DOCSTRING_ARG_REGEX = re.compile(
+    rf"^:param {ARG_NAME_SUBREGEX}:[ \t]+{ARG_DESCRIPTION_SUBREGEX}[ \t]*(?::type [^\s:]+:[ \t]+{ARG_TYPE_SUBREGEX})?",
```

Comment:
Yeah, it's not needed for this one, but as the other regexes optionally match types, I had it there for consistency.

Since there is no intention to use the match typed, I can remove it. :+1: